repository/src/main/java/org/apache/atlas/repository/graph/GraphHelper.java

```diff
@@ -1891,8 +1891,8 @@ public static String getDelimitedClassificationNames(Collection<String> classifi
      * @param parentEdgeLabel Edge label of parent
      * @return Iterator of children vertices
      */
-    public static Iterator<AtlasVertex> getActiveParentVertices(AtlasVertex vertex, String parentEdgeLabel) throws AtlasBaseException {
-        return getActiveVertices(vertex, parentEdgeLabel, AtlasEdgeDirection.IN);
+    public static Iterator<AtlasVertex> getActiveParentVertices(AtlasVertex vertex, String... parentEdgeLabel) throws AtlasBaseException {
```

Comment:
Remove this change

Reply:
if we add this line we dont need to create separate new function